CharlieBackend.Data/ApplicationContext.cs

```diff
@@ -124,6 +124,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     .HasComment("salt has been set to not null")
                     .HasCharSet("utf8mb4")
                     .HasCollation("utf8mb4_0900_ai_ci");
+
+                entity.Property(e => e.ForgotPasswordToken)
+                    .HasColumnName("forgot_password_token")
+                    .HasColumnType("varchar(100)")
+                    .HasComment("token for resetting password")
+                    .HasCharSet("utf8mb4")
+                    .HasCollation("utf8mb4_0900_ai_ci");
+
+                entity.Property(e => e.ForgotTokenGenDate)
+                    .HasColumnName("forgot_token_gen_date")
```

Comment:
can you do column name shorter maybe?

Reply:
> can you do column name shorter maybe?

No, I cant, because @SnipeFrost will kill me for that.

Reply:
I mean not shorten, but change the name as a whole